benchmarks/pom.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.crate</groupId>
+    <artifactId>crate</artifactId>
+    <version>5.5.0</version>
```

Comment:
Do I get it right, that we will be changing this version on each feature release? Is it an option to create some variable and refer to it everywhere we use hardcoded version?

Reply:
Maven provides a command to bump it across all modules: `mvn versions:set -DnewVersion=5.6.0`. But good point. I'll need to update the release docs/steps